src/documentation/pages/components/icon/examples/_base.vue

```diff
@@ -0,0 +1,40 @@
+<template lang="pug">
+<ComponentExample title="Base" id="base" padding="-p--0" :tabs="exampleTabs">
+  .-p--3(slot="example")
+    chi-icon(icon='atom')
+  <pre class="language-html" slot="code-webcomponent">
+    <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+  </pre>
+  <pre class="language-html" slot="code-htmlblueprint">
+    <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+  </pre>
+</ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({
+  data: () => {
+    return {
+      exampleTabs: [
+        {
+          disabled: true,
+          id: 'webcomponent',
+          label: 'Web component',
+        },
+        {
+          active: true,
+          id: 'htmlblueprint',
+          label: 'HTML blueprint',
+        },
+      ],
+      codeSnippets: {
+        webcomponent: ``,
```

Comment:
@soujanreddy web component is disabled and it is not accurated. Please review Chi icon doc.